CMakeLists.txt

```diff
@@ -277,7 +277,7 @@ ENDIF ()
 
 ### Open Space Toolkit ▸ Physics
 
-FIND_PACKAGE ("OpenSpaceToolkitPhysics" "0.2" REQUIRED)
+FIND_PACKAGE ("OpenSpaceToolkitPhysics" "0.8.1" REQUIRED)
```

Comment:
So I think in [theory](https://cmake.org/cmake/help/latest/command/find_package.html) and our usage so far (provided that we have been using minor versions bump only), should make it so that all versions have been compatible with what is here nah? Also it has been working in practice. idea for later: we could use an env variable to feed both the `Dockerfile` and the `CMakeLists` if we wanted

Reply:
Yeah, no need to lock everything here otherwise it's a pain to maintain :)